packages/circuits/circuits/qr_verify.circom

```diff
@@ -60,11 +57,16 @@ template QR_Verify(n, k, len) {
 
     extractor.data <== padded_message;
     extractor.selector <== selector;
-    extractor.reveal_timestamp <== reveal_timestamp;
 
     reveal_data <== extractor.reveal_data;
-    timestamp <== extractor.timestamp; 
     four_digit <== extractor.four_digit;
+
+    // Output the timestamp rounded to nearest hour
+    component date_to_timestamp = DateStringToTimestamp(2030, 1, 0, 0);
```

Comment:
@vuvoth I have removed the `reveal_timestamp` input and is always outputting formatted timestamp. Do you think we need to a conditional on this?
If we do, I think this should be a template param, so we can skip the conversion all together.

Reply:
We don't the way developer use timestamp in their application. It's good if we still keep it like the option.